src/hmi_stub/main.cc

```diff
@@ -165,6 +165,21 @@ int main(int argc, char *argv[]) {
       } else if (buffer == "fetchmetadata" || buffer == "fetchmeta") {
         aktualizr.FetchMetadata();
       } else if (buffer == "download" || buffer == "startdownload") {
+        std::thread([&aktualizr] {
+          while (int c = getchar()) {
+            if (c == 'p') {
+              std::cout << "pausing\n";
+              aktualizr.Pause();
+              break;
+            } else if (c == 'r') {
+              std::cout << "resuming\n";
+              aktualizr.Resume();
+              break;
+            }
+          }
+        })
+            .detach();
+
```

Comment:
This presumably will also need to break once the download is complete, right? Or is that automatic in some way that I'm not seeing?

I'd originally assumed you'd want to base this PR off of the API PR that we've been working on, but this is actually simpler than I expected, so maybe it is fine as it is. That other PR still isn't finished but I hopefully will get it done soon.